MonoGame.Framework.Content.Pipeline/Graphics/GraphicsUtil.cs

```diff
@@ -159,7 +167,9 @@ public static void CompressDxt(GraphicsProfile profile, TextureContent content,
 
             if (alphaRange == AlphaRange.Opaque)
                 Compress(typeof(Dxt1BitmapContent), content, generateMipMaps);
-            else if (alphaRange == AlphaRange.Cutout || sharpAlpha)
+            else if (sharpAlpha)
+                CompressFontDXT3(content, generateMipMaps);
```

Comment:
I first though I'd use 'isSpriteFont' because this compression only gives four levels of grey, including black and white, so it's not really useful for other greyscale textures unless they had very specific needs, i.e. a logo or similar.